### ISSUE TITLE
|noembed text displayed in preview mode

### ISSUE THREAD

**Issue by claremacrae** [samwarnick/obsidian-simple-embeds#19]
_Sorry, me again!_ 😄 

A small cosmetic issue...

For the first time today I tried using `|noembed` - and found it was displayed in preview mode.

Source:

```
[Reset Pebble watch|noembed](https://www.youtube.com/watch?v=xv_EkeJi1-A)
```

Result in preview, with Simple Embeds 1.6.2

![image](https://user-images.githubusercontent.com/4840096/147961820-a77cbd9c-32e4-49d9-b36e-ae784bd10253.png)

From looking at the code, I suspect this may be the result of the fix for #18 - ff220aa5f4dd8dcef1bde22adfba8888fb270d01

**Comment by samwarnick** [comment]
Ugh, Mondays. Missed that. Need to get my test cases in order.

Should have a fix for it soon—just double checking things. Thanks for reporting.

**Comment by samwarnick** [comment]
Should be fixed in [1.6.3](https://github.com/samwarnick/obsidian-simple-embeds/releases/tag/1.6.3).

**Comment by claremacrae** [comment]
Confirmed - after updating and restarting, both the things I had reported are perfect in Preview:

![image](https://user-images.githubusercontent.com/4840096/147989791-6c4056a1-ca97-4780-83b6-14c9e8ddeef5.png)

Many thanks ❤️
